parser/apartment.py

```diff
@@ -5,7 +5,9 @@
 class Apartment:
```

Comment:
You should make it NamedTuple or dataclass

Reply:
done